movai_core_shared/envvars.py

```diff
@@ -139,7 +139,7 @@
 
 REST_SCOPES = (
     "(Callback|Form|Flow|Node|GraphicScene|Package|StateMachine|Layout|Annotation|Application|"
-    "Configuration|SharedDataTemplate|SharedDataEntry|TaskTemplate|TaskEntry)"
+    "Configuration|SharedDataTemplate|SharedDataEntry|TaskTemplate|TaskEntry|Translation)"
```

Comment:
I have to say, it feel weird to see this list here. It's not like it's a *configuration*, one can't just add a new item here if it's not supported by the DAL, but we don't control the DAL version here.

What if we moved this to the backend repo? There we can set the minimum DAL version that supports the new scope, so that there's no possibility that the backend adds an endpoint that isn't actually supported.

Reply:
Moved to dal